package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-commercetools",
-  "version": "4.0.2",
+  "version": "5.0.0",
```

Comment:
Because of the major eslint version bumps, we should bump a major version as well